tox.ini

```diff
@@ -30,5 +31,10 @@ exclude = .tox,.eggs
 show-source = true
 ignore = D100,D101,D102,D103,D104
 
+[testenv:black]
+basepython = python3.6
+deps = black
+commands = black --check kojak/
```

Comment:
You need to add black to the `test-requirements.txt` file else black is not exist in the environment and this command will fail